packages/sanity/src/core/comments/index.ts

```diff
@@ -1,10 +1,33 @@
-export * from './components'
-export * from './constants'
-export * from './context'
-export * from './helpers'
-export * from './hooks'
-export * from './store'
+// this file exports the public api for comments (ends up in the sanity package) – be mindful of what gets re-exported
```

Comment:
I opted to handle hand-picked exports at this level instead of in the root  index.ts, as it is a bit easier to see whats available from each sub-directory.

This is done to expose as little as possible, while still allowing Create to reuse as much as possible.

Reply:
appreciate this comment 👍🏼  also aligns with our goal to avoid barrel exports